### PR TITLE
vala-panel: update to 0.5.0

### PR DESCRIPTION
**Pull request by joshuakraemer**

### CODE REVIEW

File: srcpkgs/vala-panel/template [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
+version=0.5.0
 revision=1
-build_style=cmake
-hostmakedepends="gettext pkg-config vala glib-devel"
-makedepends="vala-devel cmake-vala gtk+3-devel"
-short_desc="GTK+3 desktop panel inspired by LXPanel"
-maintainer="John Eira <john.eira@web.de>"
+build_style=meson
+configure_args="$(vopt_if wayland -Dplatforms=x11,layer-shell -Dplatforms=x11)"
+hostmakedepends="vala pkg-config cmake gettext"
```
Comment (ndowens) [inline srcpkgs/vala-panel/template:7]: Re-add glib-devel in host and I think that will fix the failing builds.

Comment (joshuakraemer) [reply srcpkgs/vala-panel/template]: Thanks, you are right, moving glib-devel to hostmakedepends fixed it.

File: srcpkgs/vala-panel/template [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+checksum=af7dd95a5dee6701fc1cbbdbd479a42a8e3b0991b2ce939d73de757c906c61d2
+
+# Package build options
+build_options="wayland"
+build_options_default="wayland"
 
 vala-panel-devel_package() {
 	short_desc+=" - development files"
-	depends="vala-panel-${version}_${revision}"
+	depends="${sourcepkg}-${version}_${revision}"
```
Comment (ericonr) [inline srcpkgs/vala-panel/template:22]: `-$[version}` should be `>=${version}`

File: srcpkgs/vala-panel/template [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
-hostmakedepends="gettext pkg-config vala glib-devel"
-makedepends="vala-devel cmake-vala gtk+3-devel"
-short_desc="GTK+3 desktop panel inspired by LXPanel"
-maintainer="John Eira <john.eira@web.de>"
+build_style=meson
+configure_args="$(vopt_if wayland -Dplatforms=x11,layer-shell -Dplatforms=x11)"
+hostmakedepends="vala pkg-config cmake gettext glib-devel"
+makedepends="gtk+3-devel libwnck-devel $(vopt_if wayland gtk-layer-shell-devel)"
+short_desc="GTK+3 desktop panel forked from LXPanel"
+maintainer="Joshua Krämer <joshua@kraemer.link>"
```
Comment (ericonr) [inline srcpkgs/vala-panel/template:10]: Have you contacted the previous maintainer to see if they are ok with this? @purpurrot

**Comment by sgn** [comment]
```
FAILED: app/vala-panel 
cc  -o app/vala-panel app/vala-panel.p/meson-generated_.._app.c.o app/vala-panel.p/application.c.o app/vala-panel.p/vala-panel-platform-standalone-x11.c.o -flto -Wl,--as-needed -Wl,--no-undefined -Wl,--whole-archive -Wl,--start-group util/gtk/libutil-gtk.a -Wl,--no-whole-archive -fstack-clash-protection -D_FORTIFY_SOURCE=2 -mtune=generic -O2 -pipe -fno-common -g -Wl,-z,relro -Wl,-z,now -Wl,--as-needed ui/libvalapanel.so.0.5.0 util/libutil.a /usr/lib/libgio-2.0.so /usr/lib/libgobject-2.0.so /usr/lib/libglib-2.0.so -Wl,--export-dynamic /usr/lib/libgmodule-2.0.so -pthread /usr/lib/libgtk-3.so /usr/lib/libgdk-3.so /usr/lib/libpangocairo-1.0.so /usr/lib/libpango-1.0.so /usr/lib/libharfbuzz.so /usr/lib/libatk-1.0.so /usr/lib/libcairo-gobject.so /usr/lib/libcairo.so /usr/lib/libgdk_pixbuf-2.0.so -lm -Wl,--end-group '-Wl,-rpath,$ORIGIN/../ui' -Wl,-rpath-link,/builddir/vala-panel-0.5.0/build/ui
```

**Comment by joshuakraemer** [comment]
@sgn, thank you for your additions. Please feel free to change the maintainer to yourself or keep @purpurrot if he responds.

File: srcpkgs/vala-panel/template (hunk 3 above)
Comment (joshuakraemer) [reply srcpkgs/vala-panel/template:10]: I wrote an email to him on 2020-11-07, but he did not answer. He also had no activity in his GitHub account during the last year. I thus assumed that he is no longer interested in maintaining the package.

Comment (joshuakraemer) [reply srcpkgs/vala-panel/template:10]: I wrote an email to him on 2020-11-07, but he did not answer. Additionaly, there was no activity on his GitHub account during the last year. I thus assumed that he is no longer interested in maintaining the package.

Comment (ericonr) [reply srcpkgs/vala-panel/template:10]: Ok, that's fair.

**Pull request closed without merging**